src/commands/infra/generate.ts

```diff
@@ -244,22 +243,51 @@ export const validateRemoteSource = async (
   logger.info(
     `Checking if source repo: ${safeLoggingUrl} has been already cloned to: ${sourcePath}.`
   );
-
   try {
-    // Check if .git folder exists in ${sourcePath}, if not, then clone
-    // if already cloned, 'git pull'
-    if (fs.existsSync(path.join(sourcePath, ".git"))) {
-      await gitFetchPull(sourcePath, safeLoggingUrl);
-    } else {
-      await gitClone(source, sourcePath);
-    }
+    // Clone remote source
+    await infraCommon.repoClone(sourcePath, source);
```

Comment:
we have await `infraCommon` and await `gitCheckout` function calls in this `validateRemoteSource` function hence it may take more than 5 secs to execute this command on a slow network. Then this issue, https://github.com/microsoft/bedrock/issues/935 will resurface. One way to get around this is to mock `infraCommon.repoClone`. thanks

Reply:
Removed function